spec/pb_kits/playbook/kits/caption_spec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require_relative "../../../../app/pb_kits/playbook/pb_caption/caption"
```

Comment:
When loading all the files, there were various load order issues because of the non-converted kits.

I think we'd have to do something like this commit otherwise. https://github.com/powerhome/playbook/pull/301/files/14d4702e4557b516652b5dd6fc25a051f4eaec76..95001d9548e56eb019c5107d2051537efaffcd6e#diff-93830fa29d616f7c87903d08b5b1b29a

I'm open to moving it into sorted directory load in the `spec_helper`. It'd be much nicer.